### ISSUE TITLE
Getting "token parse fail" when trying to run Nitter on a VM

### ISSUE THREAD

**Issue by ritcher** [zedeus/nitter#196]
When I try to run Nitter on a VM, I get the following output:

```
Starting Nitter at http://nitter.net
Connected to Redis at localhost:6379
token parse fail
```

When analyzing the code from [src/tokens.nim#L19](https://github.com/zedeus/nitter/blob/master/src/tokens.nim#L19), I see that Nitter makes a GET request to `https://twitter.com` in order to obtain the value of the `gt` parameter from the body of the response.

When running `curl https://twitter.com | grep gt=` on my personal machine, I get the following output:

```
document.cookie = decodeURIComponent("gt=1274235661885808640; Max-Age=10800; Domain=.twitter.com; Path=/; Secure");
```

But if I run it on my VM, I don't get any output.

Is Twitter somehow blocking requests made through the VM? If so, is there any way to circumvent this?

This is the full HTML document that I receive when making the request for `twitter.com` through the VM: [índex.html](https://bin.snopyta.org/?635e0af92238492a#7kGxexZCgKMScDFNeG7Sk8qqPRZp7NViQpZ8RhCXcGiV)

**Comment by zedeus** [comment]
That's weird, seems like you got temporarily rate limited, which only lasts up to 30 minutes. Does it work now?

**Comment by ritcher** [comment]
No, I receive practically the same response for each new request.

The value of the x-rate-limit-reset returned in the response is always reset to a new value, even if I wait for the previous limitation to expire.

**Comment by zedeus** [comment]
Is the VM running on the same machine, or rather do they have the same public IP?

**Comment by ritcher** [comment]
VMs are not running on the same machine. They also don't have the same public IP address.

I'm not sure yet, but it seems that Twitter is blocking all requests coming from addresses on the AWS network (which is the provider I'm currently using).

I tried to associate different public IP addresses with the VM and try to access Twitter through it, but to my surprise, all requests I made got rate limited.
